### ISSUE TITLE
Question about  training  fastspeech2 with v1.yaml vs y2.yaml

### ISSUE THREAD

**Issue by JunjunCui** [TensorSpeech/TensorFlowTTS#422]
Hi~ Thanks for your contribution. It's a real meaningful work.
I trained a fastspeech2 model on own dataset for Chinese language.
I trained well on v2.yaml,  but I couldn't converge good on v1.yaml，this is their loss trajectory.
Would you mind suggesting me to improve the v1 fastspeech2 .
![image](https://user-images.githubusercontent.com/37209860/101600924-f354d480-3a36-11eb-983c-c9d58b52d770.png)

**Comment by dathudeptrai** [comment]
> Hi~ Thanks for your contribution. It's a real meaningful work.
> I trained a fastspeech2 model on own dataset for Chinese language.
> I trained well on v2.yaml, but I couldn't converge good on v1.yaml，this is their loss trajectory.
> Would you mind suggesting me to improve the v1 fastspeech2 .
> ![image](https://user-images.githubusercontent.com/37209860/101600924-f354d480-3a36-11eb-983c-c9d58b52d770.png)

i'm not sure what is a problem but the `v` should be better than `v2` :)).

**Comment by JunjunCui** [comment]
> > Hi~ Thanks for your contribution. It's a real meaningful work.
> > I trained a fastspeech2 model on own dataset for Chinese language.
> > I trained well on v2.yaml, but I couldn't converge good on v1.yaml，this is their loss trajectory.
> > Would you mind suggesting me to improve the v1 fastspeech2 .
> > ![image](https://user-images.githubusercontent.com/37209860/101600924-f354d480-3a36-11eb-983c-c9d58b52d770.png)
> 
> i'm not sure what is a problem but the `v` should be better than `v2` :)). But if v2 is better in ur case, so let use `v2` and just don't care about `v1` :))

Thanks for your reply. There is a little badcase in my  results on V2, so I want to try V1, but the convergence is worse.

**Comment by dathudeptrai** [comment]
@JunjunCui try to use smaller initalize learning rate for v1.

**Comment by JunjunCui** [comment]
> @JunjunCui try to use smaller initalize learning rate for v1.

Thanks! i will try it

**Comment by JunjunCui** [comment]
@dathudeptra I used a smaller initial learning rate on V1 and got a good convergence, thank you!!
![image](https://user-images.githubusercontent.com/37209860/101711837-5003ce00-3acf-11eb-9831-42ec76e2affc.png)

**Comment by dathudeptrai** [comment]
@JunjunCui :D
